public/app/features/backup/components/StorageLocations/StorageLocationsActions/StorageLocationsActions.tsx

```diff
@@ -4,12 +4,12 @@ import { DBIcon } from '../../DBIcon';
 import { StorageLocatationsActionProps } from './StorageLocationsActions.types';
 import { getStyles } from './StorageLocationsActions.styles';
 
-export const StorageLocationsActions: FC<StorageLocatationsActionProps> = ({ location }) => {
+export const StorageLocationsActions: FC<StorageLocatationsActionProps> = ({ location, onUpdate }) => {
   const styles = useStyles(getStyles);
 
   return (
     <div className={styles.actionsWrapper}>
-      <DBIcon type="edit" data-qa="edit-storage-location-button" role="button" />
+      <DBIcon type="edit" data-qa="save-storage-location-button" role="button" onClick={() => onUpdate(location)} />
```

Comment:
Please, avoid using arrow functions as event handlers (see the note [here](https://reactjs.org/docs/faq-functions.html#arrow-function-in-render)). Create an external function, in the body of the function component if needed, and use that one.

Reply:
The line immediately below that:
```
Is it OK to use arrow functions in render methods?
Generally speaking, yes, it is OK, and it is often the easiest way to pass parameters to callback functions.
```

I have to pass the location here and this doesn't seem to be critical performance-wise.
Is it any better to curry the function for such simple case?

Reply:
If you had to do that in a loop and pass a parameter from the loop, that yes, it would be the easiest way. In this case though it's better to put it outside, at least for consistency (since we're following this practice elsewhere). As a general consideration, keeping the functions outside the rendering makes them more readable and easily extendable (without requiring extra effort and even improving performance slightly).

Reply:
Ok, got it 👍